packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -238,9 +238,7 @@ export class ProjectActions {
 
     this.ctx.coreData.currentTestingType = testingType
 
-    this.ctx.browser.setBrowserStatus('opening')
     await this.api.launchProject(browser, activeSpec ?? emptySpec, options)
-    this.ctx.browser.setBrowserStatus('open')
```

Comment:
These status updates were moved into the server so that the launchpad state gets updated no matter how the browser is opened.